src/app/pages/login/login.component.html

```diff
@@ -47,32 +47,47 @@
                 </style>
             </div>
 
-            <img src="assets/logo-docks-150x.png" class="img-fluid" alt="Responsive image" width="150px" id="logo">
 
-            <div class="card border-secondary" align="left">
-                <div class="card-body">
-                    <h3 class="card-title" align="center">Login</h3>
-                    <div class="form-group">
-                        <label for="username">Username</label>
-                        <input type="text" class="form-control" [ngClass]="{'is-invalid'  : usernameError}" placeholder="Username" aria-label="Username" id="username" #username>
-                    </div>
-                    <div class="form-group">
-                        <label for="password">Password</label>
-                        <input type="password" class="form-control" [ngClass]="{'is-invalid'  : passwordError}" placeholder="Password" aria-label="Password" id="password" (keyup.enter)="login(username.value, password.value)"
-                               #password>
-                    </div>
+        </div>
 
-                    <div *ngIf="statusMessage !== ''">
-                        <ngb-alert type="danger" (close)="statusMessage = ''">{{statusMessage}}</ngb-alert>
-                    </div>
+    </div>
+</div>
+<div class="row">
+    <div class="col-md-4 offset-md-3 col-sm-6 offset-sm-3 col-xs-12 image">
+        <img src="assets/logo-docks-150x.png" class="img-fluid" alt="Responsive image" width="150px" id="logo">
+    </div>
+</div>
 
-                    <div align="center">
-                        <button type="button" id="loginButton" class="btn btn-success btn-block" [disabled]="busy" (click)="login(username.value, password.value)">Login</button>
+<div class="row">
+    <div class="col-md-4 offset-md-3 col-sm-6 offset-sm-3 col-xs-12">
+        <div class="card border-secondary" align="left">
+            <div class="card-body">
+                <h3 class="card-title" align="center">Login</h3>
+                <div class="form-group">
+                    <label for="username">Username</label>
+                    <input type="text" class="form-control" placeholder="Username" aria-label="Username" id="username" #username>
+                    <div [hidden]="!usernameError" class="alert alert-danger">
+                        Username is required
                     </div>
+                </div>
+                <div class="form-group">
+                    <label for="password">Password</label>
+                    <input type="password" class="form-control" placeholder="Password" aria-label="Password" id="password" (keyup.enter)="login(username.value, password.value)"
+                        #password>
+                    <div [hidden]="!passwordError" class="alert alert-danger">
+                        Password is required
+                    </div>
+                </div>
+
+                <div *ngIf="statusMessage !== ''">
```

Comment:
Noticed that I also didn't remove this in develop yet. This can be removed and the statusMessage can be removed in the ts file aswel